_dashboards/discover/multi-data-sources.md

```diff
@@ -7,42 +7,31 @@ nav_order: 20
 
 # Adding multiple data sources
 
-OpenSearch Dashboards allows you to dynamically manage data sources, create index patterns based on those data sources, run queries against a specific data source, and combine visualizations in one dashboard.
-
-In this tutorial we provide the steps for enabling the `data_source` setting in Dashboards; adding credentials, data source connections, and index patterns; and combining visualizations in a single dashboard.
-
-## Enabling the multiple data sources feature
-
-This tutorial uses a preconfigured data source and index pattern, and you aren’t required to configure settings. However, you’ll need to enable the `data_source` setting in the configuration file before before getting started with exploring this feature.
-{: .note }
+OpenSearch Dashboards allows you to dynamically manage data sources, create index patterns based on those data sources, run queries against a specific data source, and combine visualizations in one dashboard. In this tutorial we provide the steps for enabling the `data_source` setting in Dashboards; adding credentials, data source connections, and index patterns; and combining visualizations in a single dashboard.
```

Comment:
According to the new style guidance, we should say something like, "With OpenSearch Dashboards, you can dynamically manage data sources, create index patterns based on those data sources, run queries against a specific data source, and combine visualizations in a single dashboard."     (use phrases such as “With this feature, you can...” or “Use this feature to...” instead of saying a feature allows, enables, or lets the user do something.)

Reply:
Revised throughout to follow style guide.